lib/aweplug/extensions/kramdown_quickstart.rb

```diff
@@ -76,11 +81,15 @@ def initialize opts = {}
         def execute site
           cache = Aweplug::Cache.default site # default cache shouldn't matter here
 
-          Parallel.each(Dir["#{@repo}/*/README.md"], :in_threads => (site.build_threads || 0)) do |file|
-            searchisko = Aweplug::Helpers::Searchisko.new({:base_url => site.dcp_base_url, 
-                                                           :authenticate => true, 
-                                                           :searchisko_username => ENV['dcp_user'], 
-                                                           :searchisko_password => ENV['dcp_password'], 
+          path_to_readme = []
+          Find.find("#{@repo}") do |path|
+            path_to_readme << path if path =~ /(.*\/README.md)|(.*\/Readme.md)/
```

Comment:
Do we want to catch `readme.md` as well? Or any other case combination for that matter? It may also be better to pull out the regexp creation from the loop to avoid creating it over and over again.